mozilla-mobile/android-components/components/feature/contextmenu/src/main/res/values/strings.xml

```diff
@@ -30,6 +30,14 @@
     <string name="mozac_feature_contextmenu_snackbar_action_switch">Switch</string>
     <!-- Text for context menu item to open the link in an external app. -->
     <string name="mozac_feature_contextmenu_open_link_in_external_app">Open link in external app</string>
+    <!-- Text for context menu item to share the email with another app. -->
+    <string name="mozac_feature_contextmenu_share_email_address">Share email address</string>
+    <!-- Text for context menu item to copy the email address to the clipboard. -->
+    <string name="mozac_feature_contextmenu_copy_email_address">Copy email address</string>
+    <!-- Text for confirmation "snackbar" shown after copying a email address to the clipboard. -->
+    <string name="mozac_feature_contextmenu_snackbar_email_address_copied">Email address copied to clipboard</string>
+    <!-- Text for context menu item to add to contact. -->
```

Comment:
@rocketsroger: The more I read this string with my non-US mind, the more I think there is a small risk for confusion here, on whether "contact" is a noun or a verb.
 To avoid any confusion in our translations, could you please change the comment by adding an "a", so: "Text for context menu item to add to **a** contact"

Other than that this is good to go

Reply:
Sure, I'll update.  Thanks for catching this.

Reply:
Change merged.  https://github.com/mozilla-mobile/android-components/pull/8044.  Thanks!

Reply:
Thanks for the quick turnaround!